plugins.md

```diff
@@ -2965,6 +2977,33 @@ External services that transform an address or the name of a place into latitude
 
 
 
+### External framework controls
```

Comment:
I don't think we need yet another plufgin category. I'd say this would fit under the current [frameworks & build systems](http://leafletjs.com/plugins.html#frameworks--build-systems) category.